kairon/api/app/routers/bot/data.py

```
@@ -281,6 +281,24 @@ async def get_collection_data(
                                                                  key=key, value=value))}
 
 
+@router.get("/collection/{collection_name}/filter", response_model=Response)
+async def get_collection_data_with_timestamp(
+        collection_name: str,
+        filters = Query(default='{}'),
```

Ruff (0.8.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)